modules/rosa-deleting-aws-resources-cli.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa-sts-deleting-cluster.adoc
+
+[id="rosa-deleting-aws-resources-cli_{context}"]
+= Deleting the AWS resources by using the CLI
+
+After deleting a {product-title} (ROSA) cluster, you can delete the AWS Security Token Service (STS) resources by using the CLI.
+
+[IMPORTANT]
+====
+Account-wide Identity Access Management (IAM) roles and policies might be used by other ROSA clusters in the same account. You must only remove the resources if they are not required by other clusters.
+====
+
+.Prerequisites
+
+* You have deleted your ROSA cluster. For more information, see the _Deleting a cluster_ section.
+
+.Procedure
+
+. Delete the OpenID Connect (OIDC) provider that you created for Operator authentication in your cluster:
++
+[source,terminal]
+----
+$ aws iam delete-open-id-connect-provider --open-id-connect-provider-arn <oidc_provider_arn> <1>
+----
+<1> Replace `<oidc_provider_arn>` with the Amazon Resource Name (ARN) of the OpenID Connect (OIDC) resource that you created to authenticate the cluster Operators. You can list the OIDC providers in your account by running `$ aws iam list-open-id-connect-providers`.
```

Comment:
For the commands like `aws iam list-open-id-connect-providers` ,there is a stop followed which is in same line with the commands. Users may copy it accidentally or wrongly. I am wondering if we can deal with such commands in same style with others to avoid.

Reply:
Thank you for highlighting this. I have updated the structure of each of the sentences that include the list commands so that the commands are not at the end of the sentence. Each of the commands are subsequently not followed by a full stop, but instead by a space. I want to avoid adding an extra step in in the procedure for these list commands because they are just a tip really for the reader and I don't think they warrant adding several additional substeps.